gammapy/cube/models.py

```diff
@@ -340,3 +341,48 @@ def evaluate(self, lon, lat, energy):
     def copy(self):
         """A shallow copy"""
         return copy.copy(self)
+
+
+class BackgroundModel(Model):
+    """Background model
+
+    Parameters
+    ----------
+    background : `~gammapy.maps.Map`
+        Background model map
+    norm : float
+        Background normalisation
+    tilt : float
+        Additional tilt in the spectrum
+    reference : `Quantity`
+        Reference energy of the tilt.
+    """
+    def __init__(self, background, norm=1, tilt=0, reference="1 TeV"):
+
+        axis = background.geom.get_axis_by_name("energy")
+        if axis.node_type != "edges":
+            raise ValueError('Need an integrated map, energy axis node_type="edges"')
+
+        self.map = background
+        self.parameters = Parameters([
+            Parameter("norm", norm, unit=""),
+            Parameter("tilt", tilt, unit="", frozen=True),
+            Parameter("reference", reference, frozen=True),
+        ])
+
+
+    def energy_center(self):
+        """True energy axis bin centers (`~astropy.units.Quantity`)"""
+        energy_axis = self.map.geom.get_axis_by_name("energy")
+        energy = energy_axis.center
+        return energy[:, np.newaxis, np.newaxis]
+
+
+    def evaluate(self):
+        """Evaluate background model"""
+        norm = self.parameters["norm"].value
+        tilt = self.parameters["tilt"].value
+        reference = self.parameters["reference"].value
+        tilt_factor = np.power(self.energy_center()/reference, -tilt)
```

Comment:
There is the possibility that `energy_center` and `reference` are not in the same unit. This has to be handled somehow. I think the easiest way make `self.energy_center` return a `Quantity` and change the reference parameter access to `self.parameters["reference"].quantity` as well. To make sure the tilt factor is in dimensionless you can explicitly use `(energy / reference).to("")`.

Reply:
Done